packages/cyberstorm/src/components/componentStyles/Tag.module.css

```diff
@@ -17,46 +17,49 @@
   --border-color: transparent;
 }
 
-.tag:hover {
+.root:hover {
   --bg-color: var(--color-surface--0);
   --border-color: var(--color-border-3);
 }
 
 /* Different Tag sizes */
 
 .small {
-  gap: var(--size-tiny);
-  padding: var(--size-tiny) var(--text-xxs);
-  font: normal 700 var(--text-xxs) Inter, sans-serif;
-  line-height: var(--size-tiny);
+  gap: var(--gap-tiny);
+  padding: var(--padding-tiny) var(--padding-xxs);
+  font: var(--font-inter);
+  font-weight: var(--font-weight-bold);
+  font-size: var(--text-xxs);
 }
 
 .medium {
-  gap: var(--size-tiny);
-  padding: var(--size-xxs) var(--text-xs);
-  font: normal 700 var(--text-xs) Inter, sans-serif;
-  line-height: var(--text-m);
+  gap: var(--gap-tiny);
+  padding: var(--padding-xxs) var(--text-xs);
+  font: var(--font-inter);
+  font-weight: var(--font-weight-bold);
+  font-size: var(--text-xs);
 }
 
 .large {
-  gap: var(--size-xxs);
-  padding: var(--size-xs) var(--text-xs);
-  font: normal 700 var(--text-m) Inter, sans-serif;
-  line-height: var(--text-l);
+  gap: var(--gap-xxs);
+  padding: var(--padding-xs) var(--text-xs);
+  font: var(--font-inter);
+  font-weight: var(--font-weight-bold);
+  font-size: var(--text-m);
 }
 
 /* Styles for Tag's text */
-.tagLabel {
+.label {
   text-transform: uppercase;
 }
 
 /* Styles for Tag's Icons */
-.tagIcon {
+.icon {
   color: var(--color-border-3);
   line-height: var(--text-s);
 }
 
 /* Different Tag color schemes */
-.tag__default:hover > .tagIcon {
+.tag__default:hover > .icon {
   color: var(--color-red);
```

Comment:
This probably should only apply when a tag has an icon indicating it can be destructed.

Reply:
👍

Reply:
I was thinking that this could have some class related to being able to be removed, not only when having an icon -  use case being the category filters on mod list page. This can be done later though when the filter functionality is further specified.

Reply:
I think I understood it now! Made some changes

Reply:
This css rule will change all icons to red, not just the `x` one shown when `isBoolean` is true. Would be good to revisit when building that functionality. This will become relevant once the package list filtering is taken under construction.